connector-luna/src/main/java/com/datastax/oss/pulsar/source/CassandraSource.java

```diff
@@ -704,6 +705,12 @@ public MyKVRecord(ConverterAndQuery converterAndQueryFinal, CompletableFuture<Ke
             return msg;
         }
 
+        @Override
+        @SuppressWarnings("unchecked")
+        public Schema getSchema() {
+            return Schema.KeyValue(getKeySchema(), getValueSchema(), getKeyValueEncodingType());
```

Comment:
We should create a new instance every time, because schema are stateful.

Also, this method is not used by the Pulsar IO framework as this is a KVRecord.

I guess why you are adding this method, but unfortunately this is not the contract of Pulsar IO